informer/disk/disk.go

```diff
@@ -28,6 +36,14 @@ type Informer struct {
 
 // NewInformer returns an initialized Informer.
 func NewInformer() *Informer {
+	MetricName = DefaultMetric
```

Comment:
On the line of previous comment, I think you can simply make the metric name a property of the informer...